pkg/reconciler/ingress/config/istio.go

```diff
@@ -149,7 +152,7 @@ func NewIstioFromConfigMap(configMap *corev1.ConfigMap) (*Istio, error) {
 
 	var statusEnabled bool
 	if err := cm.Parse(configMap.Data,
-		cm.AsBool("enable-virtualservice-status", &statusEnabled),
+		cm.AsBool(EnableVSStatus, &statusEnabled),
 	); err != nil {
 		return nil, err
```

Comment:
fwiw, should this even be a failure mode?

Reply:
yeah good point, this is a preview feature so we can just default to false if there's an error.

Reply:
though actually since there's no logging available here i'd prefer an error here just for sanity. most people won't touch this config anyways